### DISCUSSION TITLE
Questions about renormalized 2nd-constants

### DISCUSSION THREAD

**Discussion by Grover-Hu** [ttadano/alamode#163] [Q&A]
Dear Dr. Tadano,

I have used scph to get the PREFIX.scph_dfc2.  I want to ask how can I get the renormalized 2nd-constants in ShengBTE format (FORCE_CONSTANT_2ND)? I see there are dfc2 and scph_to_qefc.py scripts. So maybe I need to use QE to calculate the original 2nd-constants and use scph_to_qefc.py script? (I am only familiar with VASP now.) 

Thank you very much.
Looking for your reply!

**Comment by vivekgusain09** [comment]
I am facing the same problem. There is  a script for Quantum Espresson format of second order IFCs but i am not able to do it for my VASP output. Were you able to do the same ? Could you please guide me on how i can get the renormalized second order IFCs and then convert them to shengbte format ? Any help is highly appreciated. Thank you in advance.

**Comment by vivekgusain09** [comment]
I have found the solution. If someone is facing the same problem you can use the following process.
1. Go to /alamode/_build/tools
2. Find executible named 'dfc2' (it is generated during the compilation process)
3. dfc2.* : Create effective harmonic force constant (ALAMODE XML) from input harmonic force constant (XML format) and PREFIX.scph_dfc2. (this line is copied from Readme.md from alamode github) 
Hope it helps!
